providers/resource/bin/graphics_card_resource.py

```diff
@@ -23,6 +23,25 @@
 import subprocess
 import shlex
 import string
+import re
+
+
+def get_ubuntu_version():
+    """Get Ubuntu release version for checking."""
+    try:
+        import distro
+        return float(distro.version())
+    except (ImportError, subprocess.CalledProcessError):
```

Comment:
What's the use case where `distro` is not available? It seems [available since 18.04](https://packages.ubuntu.com/search?keywords=python3-distro) (but I'm not sure since when it's installed by default).

Reply:
`distro` is [supported and tested on Python 3.6+](https://pypi.org/project/distro/). If py3.5 removed from tox checking, then we don't need this code.

Reply:
You're right, we still need to support Python 3.5 which is used on Ubuntu Core 16 (we still build Checkbox snaps for core 16 projects). No need to change anything here then.